adoc/admin-kubernetes-changes.adoc

```diff
@@ -0,0 +1,74 @@
+== From Kubernetes 1.17 to 1.18
+
+This documentation page lists all the behavioral changes and API deprecations that will happen from Kubernetes 1.17 to 1.18. This is going to be relevant when migrating to {productname} 4.3.x, since it will bring Kubernetes 1.18.
+
+=== Architectural changes in {productname}
```

Comment:
I am not sure this is very important to the deployers. I would rather have that as the release note in this repo, explaining in which part the hyperkube removal might impact them (mirroring of the image for airgapped).

I would remove this from here.